docs/openhabian.md

```diff
@@ -1,159 +1,144 @@
+
+As of openHABian version 1.6 and later, all Raspberry Pi models are supported as hardware.
+Anything x86 based may work or not.
+Anything else ARM based such as ODroids, OrangePis and the like may work or not.
+NAS servers such as QNAP and Synology boxes will not work.
+Support for PINEA64 was dropped in this current release.
+We strongly recommend that users choose Raspberry Pi 2, 3 or 4 systems that have 1 GB of RAM or more.
+RPi 1 and 0/0W only have a single CPU core and 512 MB of RAM.
+This can be sufficient to run a smallish openHAB setup, but it will not be enough to run a full-blown system with many bindings and memory consuming openHABian features/components such as zram, InfluxDB or Grafana.
+We do not actively prohibit installation on any hardware, including unsupported systems, but we might skip or deny to install specific extensions such as those memory hungry applications named above.
+
+Supporting hardware means testing every single patch and every release.
+There are simply too many combinations of SBCs, peripherals and OS flavors that maintainers do not have available, or, even if they did, the time to spend on the testing efforts that is required to make openHABian a reliable system.
+Let's make sure you understand the implications of these statements: it means that to run on hardware other than RPi 2/3/4 or (bare metal i.e. not virtualized) x86 Debian may work but this is **not** supported.
+
+It may work to install and run openHABian on unsupported hardware.
+If it does not, you are welcome to find out what's missing and contribute it back to the community with a GitHub pull request.
+It is sometimes simple things like a naming string.
+We'll be happy to include that in openHABian so you can use your box with openHABian unless there's a valid reason to change or remove it.
+However, that does not make your box a "supported" one as we don't have it available for our further development and testing.
+So there remains a risk that future openHABian releases will fail to work on your SBC because we changed a thing that broke support for your HW - unintentionally, however inevitably.
+
+For ARM hardware that we don't support, you can try any of the [fake hardware parameters](openhabian.md/#fake-hardware-mode) to 'simulate' RPi hardware and Raspi OS.
+If that still doesn't work for you, give [Ubuntu](https://ubuntu.com/download/iot) or [ARMbian](https://www.armbian.com/) a try.
+
+Going beyond what the RPi image provides, as a manually installed set of scripts, we support running openHABian on x86 hardware on generic Debian.
 On ARM, we only support Raspberry Pi OS.
 These are what we develop and test openHABian against.
-We do **not support Ubuntu** so no promises. We provide code "as-is", it may work or not.
-Several optional components such as WireGuard or Homegear are known to expose problems.
+We do **not** actively **support Ubuntu** so no promises but we provide code "as-is" that is known to run on there.
```

Comment:
I intentionally removed the "actively". As I had to learn there's still people to insist that wuith that in they understand it means it should work to use Ubuntu (and sooner or later show up asking for help when it does not, leading this whole paragraph ad adsurdum).